deploy/static/provider/aws/deploy-tls-termination.yaml

```diff
@@ -462,17 +462,17 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: ingress-nginx-admission
+  namespace: ingress-nginx
```

Comment:
Since we added the namespace field to the template, it will be fixed at this position.